qiime2/sdk/usage.py

```diff
@@ -1202,7 +1202,7 @@ def help(self, action: 'qiime2.sdk.usage.UsageAction'):
 
         Examples
         --------
-        >>> use.help(use.UsageAction('dummpy_plugin', 'split_ints'))
```

Comment:
dummpy ❤️ darwub

Reply:
I love this

Reply:
I may just be an under-parameterized RNN...

Reply:
i just lolled at that.